metron-platform/metron-common/src/main/java/org/apache/metron/common/utils/HDFSUtils.java

```diff
@@ -29,6 +29,16 @@
 
 public class HDFSUtils {
 
+  public static byte[] readBytes(String path) throws IOException {
+    return readBytes(new Path(path));
+  }
+
+  public static byte[] readBytes(Path inPath) throws IOException {
+    FileSystem fs = FileSystem.get(inPath.toUri(), new Configuration());
+    FSDataInputStream inputStream = fs.open(inPath);
```

Comment:
This should be in a try with resources block otherwise you're leaving streams open.

Reply:
I retract this, it originated in @mmiklavc 's PR.  I moved the comment there.